opennms-doc/guide-admin/src/asciidoc/text/minion/kafka.adoc

```diff
@@ -1,112 +0,0 @@
-
-// Allow GitHub image rendering
-:imagesdir: ../../images
-
-[[ga-minion-kafka]]
-=== Using Kafka for Sink ( Traps and Syslog)
-
-By default, _{opennms-product-name}_ uses the embedded _ActiveMQ_ broker to communicate with _Minions_.
-This broker is used for both issuing remote procedure calls (RPCs, ie. ping this host) and for transporting unsolicited messages such as SNMP traps and syslog messages.
-
-_Apache Kafka_ can be used as an alternative to _ActiveMQ_ for transporting the unsolicited messages.
```

Comment:
True for Meridian 2018, but for Horizon 23+ and Meridian 2019, Kafka can be used for both RPC and Sink.

Reply:
Can you help me here, you have added the comment for text which is removed.